src/hexo/themes/documentation/source/images/results-security.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 72 72" style="enable-background:new 0 0 72 72;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#ACADDA;}
+	.st1{fill:#2D28A1;}
+</style>
+<title>icons</title>
+<g>
+	<path class="st0" d="M31.8,21.3V10.6c0-1.2-1-2.2-2.2-2.1H14.8c-1.2,0-2.2,1-2.2,2.2v10.6C1.6,26.5-3.1,39.8,2.2,50.9
+		s18.5,15.8,29.6,10.5s15.8-18.5,10.5-29.6C40.1,27.2,36.4,23.5,31.8,21.3z M17,12.8h10.5v6.9c-3.4-0.8-7-0.8-10.5,0V12.8z
+		 M22.2,59.2c-9.9,0-17.9-8-17.9-17.9s8-17.9,17.9-17.9c9.9,0,17.9,8,17.9,17.9C40.1,51.2,32.1,59.2,22.2,59.2z"/>
+	<path class="st1" d="M19.8,41.9h4.8v7.2c0,1.3-1.1,2.4-2.4,2.4s-2.4-1.1-2.4-2.4l0,0V41.9z"/>
+	<circle class="st1" cx="22.2" cy="38.7" r="4.7"/>
+</g>
+</svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 72 72"><style>.a{fill:#acadda}.b{fill:#2d28a1}</style><path d="M31.8 21.3V10.6c0-1.2-1-2.2-2.2-2.1H14.8c-1.2 0-2.2 1-2.2 2.2v10.6C1.6 26.5-3.1 39.8 2.2 50.9s18.5 15.8 29.6 10.5 15.8-18.5 10.5-29.6c-2.2-4.6-5.9-8.3-10.5-10.5zM17 12.8h10.5v6.9c-3.4-.8-7-.8-10.5 0v-6.9zm5.2 46.4c-9.9 0-17.9-8-17.9-17.9s8-17.9 17.9-17.9c9.9 0 17.9 8 17.9 17.9 0 9.9-8 17.9-17.9 17.9z" class="a"/><path d="M19.8 41.9h4.8v7.2c0 1.3-1.1 2.4-2.4 2.4s-2.4-1.1-2.4-2.4v-7.2z" class="b"/><circle cx="22.2" cy="38.7" r="4.7" class="b"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?